sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/samples/TimeSeriesInsightsClientSample/QuerySamples.cs

```diff
@@ -74,15 +76,99 @@ await foreach (TimeSeriesPoint point in humidityEventsQueryAnalyzer.GetResultsAs
             #endregion Snippet:TimeSeriesInsightsSampleQueryEventsUsingTimeSpan
         }
 
-        private async Task RunQuerySeriesSample(TimeSeriesInsightsClient client, TimeSeriesId tsId)
+        private async Task RunQuerySeriesSampleWithPreDefinedVariables(TimeSeriesInsightsClient client, TimeSeriesId tsId)
+        {
+            // In this sample, we demonstrate querying for series data with predefined numeric variables in a predefined Time Series type
```

Comment:
Is it right to say predefined Time Series type? I see we are creating type below.

Reply:
The reason why I say it's predefined because in the context of the snippet itself, it is predefined. The snippet does not show that we are creating variables and types, rather assume that they are predefined before hand on the portal. Let me know if that makes sense or you think we should still revisit.